pkg/apis/serving/v1beta1/trained_model.go

```diff
@@ -51,7 +51,7 @@ type TrainedModelSpec struct {
 	InferenceService string `json:"inferenceService"`
 	// Predictor model spec
 	// +required
-	PredictorModel ModelSpec `json:"predictorModel"`
+	Inference ModelSpec `json:"inference"`
```

Comment:
I feel `PredictorModel` is better name or just `model`?

Reply:
Updated to "Model ModelSpec `json:"model" per our discussion.